react/tests/__snapshots__/Icon.test.tsx.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Don't think snapshot tests are really useful. Since the components have almost no logic. You rather test your util functions.

Reply:
Added tests for helpers